enterprise/frontend/src/embedding-sdk/components/private/RenderSingleCopy/RenderSingleCopy.tsx

```diff
@@ -0,0 +1,32 @@
+import { type PropsWithChildren, useEffect, useId } from "react";
+
+import { useMetabaseProviderPropsStore } from "embedding-sdk/sdk-shared/hooks/use-metabase-provider-props-store";
+
+export const RenderSingleCopy = ({ children }: PropsWithChildren) => {
```

Comment:
This component is important. Because now each SDK component renders an internal MetabaseProvider under the hood, some components rendered inside the internal MetabaseProvider must be rendered in a single copy: global styles, locale providers, error banners, etc.

It renders children only for the first mounted instance of this component (based on singleCopyWrapperIds) 

And also based on singleCopyWrapperIds, the component can dynamically observe if one of SDK components is unmounted (and its corresponding MetabaseProvider), so other copy will render children instead